src/cmd/cos_customizer/install_packages.go

```diff
@@ -55,7 +55,7 @@ func (ip *InstallPackage) Usage() string {
 
 // SetFlags implements subcommands.Command.SetFlags.
 func (ip *InstallPackage) SetFlags(f *flag.FlagSet) {
-	f.StringVar(&ip.PkgSpecDir, "pkgspec-dir", "", "Path to the directory that has the package spec.")
+	f.StringVar(&ip.PkgSpecDir, "pkgspec-url", "", "URL path that points to the package spec.")
```

Comment:
PkgSpecDir appears to truly be treated like a directory: https://github.com/GoogleCloudPlatform/cos-customizer/blob/master/src/pkg/provisioner/install_packages_step.go#L57

Are additional changes needed in install_packages_step.go? Would it make sense to implement a test for reading the pkgspec from GCS?

Also, it would be good to update the godoc in provisioner/config.go to be clearer on the valid inputs for PkgSpecDir.

Reply:
But the changes related to downloading of files from gcs is still in progress. Right now, the only change is just the renaming of the pkg-spec-dir. It should be fine for now to just update the input interface and in the second iteration we can update the rest.

Reply:
I would prefer to update the interface/documentation once the behavior is actually implemented. Otherwise, there will be unnecessary confusion (mismatch between documentation and behavior). Let's hold these changes until the behavior is implemented. If you want, we can include these changes and the implementation in the same PR.